src/include_complete.cc

```diff
@@ -106,8 +106,8 @@ void IncludeComplete::Rescan() {
     SetThreadName("scan_includes");
     Timer timer;
 
-    InsertIncludesFromDirectory(g_config->projectRoot,
-                                false /*use_angle_brackets*/);
+    // InsertIncludesFromDirectory(g_config->projectRoot,
```

Comment:
This can just be removed. (I didn't even notice this because I don't use include completion much)

https://gcc.gnu.org/onlinedocs/gcc/Directory-Options.html